File: Horario.java
import java.util.Objects;

class Horario {
    private String dias;
    private String horas;

    public Horario(String dias, String horas) {
        this.dias = dias;
        this.horas = horas;
    }

    public String getDias() { return dias; }
    public void setDias(String dias) { this.dias = dias; }
    public String getHoras() { return horas; }
    public void setHoras(String horas) { this.horas = horas; }

    @Override
    public String toString() {
        return dias + " a las " + horas;
    }
}

File: Python/horario.py
class Horario:
    def __init__(self, dias: str, horas: str):
        self.dias = dias
        self.horas = horas

    def get_dias(self) -> str:
        return self.dias

    def set_dias(self, dias: str):
        self.dias = dias

    def get_horas(self) -> str:
        return self.horas

    def set_horas(self, horas: str):
        self.horas = horas

    def __str__(self):
        return f"{self.dias} a las {self.horas}"
